client/src/component/Milestone.js

```diff
@@ -0,0 +1,20 @@
+import "./Home.js";
+
+
+document.addEventListener("DOMContentLoaded", function() {
+    // Create a module object
+    const module = {
+      name: "Final Project",
+      deadline: new Date("2023-07-08"),
+    };
+
+    // Calculate the remaining days to the deadline
+    const currentDate = new Date();
+    const timeDiff = module.deadline.getTime() - currentDate.getTime();
+    const remainingDays = Math.ceil(timeDiff / (1000 * 60 * 60 * 24));
+
+    // Update the HTML elements with module information
+    document.getElementById("module-name").textContent += module.name;
+    document.getElementById("deadline").textContent += module.deadline.toDateString();
+    document.getElementById("remaining-days").textContent += remainingDays + " days to deadline";
+  });
```

Comment:
This is vanilla JavaScrip, could you write it as a react component please. You can look at the other components to get started.

Reply:
done